spec/controllers/skins_controller_spec.rb

```diff
@@ -32,7 +32,7 @@
         it_behaves_like "unauthorized admin cannot edit"
       end
 
-      %w[board communications docs open_doors policy_and_abuse support tag_wrangling translation].each do |role|
+      %w[board communications docs elections open_doors policy_and_abuse support tag_wrangling translation].each do |role|
```

Comment:
Does `(Admin::VALID_ROLES - %w[superadmin support]).each do |role|` work here and/or on line 107?